NOT_SUPPORTED_USE_CASES.md

```diff
@@ -0,0 +1,45 @@
+# Rare cases in which backups are not supported by BBQ
+
+In case of some rare circumstances BBQ is not able to verify backup last update time. Such situation results in not creating most up to date backup or creating wrong backup.
+
+## `lastModifiedTime` changed to past value when chunk of data is deleted 
+
+When part of BigQuery table/partition is deleted and deleted part was whole chunk of BigQuery internal storeage, then `lastModifiedTime` is max value taken from other chunks which might be earlier than `lastModifiedTime` of deleted chunk.
+
+#### Prerequisites
+* Part of data needs to be deleted
+* Deleted part is whole BigQuery internal chunk of data
+* There is no further changes to the partition
+
+#### Result
+* New backup is not created. BBQ stores previous version which includes deleted data
+
+## Data stored in [__UNPARTITIONED__ partition](https://cloud.google.com/bigquery/docs/querying-partitioned-tables#ingestion-time_partitioned_tables_unpartitioned_partition) for a long time
+
+When data is slowly streamed to BigQuery partitioned table to partitions outside of the 10 day window (7 day past and 3 day ahead), then that data might be moved from `__UNPARTITIONED__` partition into correct one after several hours. `lastModifiedTime` is set to time of streaming (which might be several hours ago), not when data was moved between partitions. This results in not backing up new part of data, because BBQ looks at time of last backup.
+
+#### Prerequisites
+* Ingestion-time partitioned table
+* Data is streamed to the BigQuery without specifying `partitionId` or outside of the 10 day window (7 day past and 3 day ahead)
```

Comment:
just for curiosity, how data could be streamed outside 10 day window in case of ingestion-time partitioning? isn't ingestion-time set on BQ side , not on streamer side?

Reply:
Updated